cli/gh-pr-filters.go

```diff
@@ -89,14 +89,15 @@ func GetFilterForMilestone(milestoneRaw string) *Filter {
 		PR: func(pr github.PullRequest) (bool, error) {
 			milestone := pr.GetMilestone().GetTitle()
 
-			// nolint:gocritic
+			//nolint:gocritic
```

Comment:
cli/gh-pr-filters.go:92:4: directive `// nolint:gocritic` should be written without leading space as `//nolint:gocritic` (nolintlint)
```
                        // nolint:gocritic
                        ^
```